### PR TITLE
584 - article store env values

### PR DESCRIPTION
**Pull request by Cooryd**

### CODE REVIEW

File: helm/libero-editor/Chart.yaml [hunk 1]
```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "latest"
 description: eLife specific deployment of Libero Editor
 name: libero-editor
-version: 0.0.8
+version: 0.0.9
```
Comment (Cooryd) [inline helm/libero-editor/Chart.yaml:5]: Worried this might break something with current staging 🤷‍♂️

Comment (NuclearRedeye) [reply helm/libero-editor/Chart.yaml:5]: Editor isn't a critical system, so no need to panic if things don't work first time :-)

File: helm/libero-editor/templates/deployment-article-store.yaml [hunk 2]
```diff
@@ -36,3 +36,31 @@ spec:
...
+            - name: DB_NAME
+              value: {{ .Values.database.name | quote }}
+            - name: DB_SSL_VALIDATE
+              value: {{ .Values.database.sslValidate | quote }}
+            - name: AWS_REGION
+              value: {{ .Values.aws.region | quote }}
+            - name: AWS_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.aws.secret | quote }}
```
Comment (NuclearRedeye) [inline helm/libero-editor/templates/deployment-article-store.yaml:54]: Are `AWS_ACCESS_KEY` and `AWS_SECRET_ACCESS_KEY` supposed to be the same? The are both set to the value of `Values.aws.secret`?

**Review by NuclearRedeye — APPROVED** [review]
On thing you might want to double check, but other than that seems fine to me.

File: helm/libero-editor/templates/deployment-article-store.yaml (hunk 2 above)
Comment (Cooryd) [reply helm/libero-editor/templates/deployment-article-store.yaml:54]: They're related so they live in the same kubeseal file. `Values.aws.secret` references the file and the key points to the value inside

**Pull request merged**